build.py

```diff
@@ -69,11 +70,11 @@ def main():
         logger.info("Now calling build-sysroot.py. See ya later!")
         args = [sys.executable, "build-sysroot.py", "--build", "python", "pyqt5", "sip", "--enable-dynamic-loading", "--sysroot=root"]
         if _WIN32:
-            del args[3]
```

Comment:
we actually need to pass the "python" value to the --build option to make it do its thing